snackbar_presenter/lib/snackbar_presenter.dart

```diff
@@ -0,0 +1,76 @@
+library snackbar_presenter;
+
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+
+const _successColor = Color(0xff4dd662);
+const _errorColor = Color(0xffed6e73);
+const _separationDistance = 12.0;
+
+class SnackBarPresenter {
+  const SnackBarPresenter._();
+
+  /// Present an information [SnackBar] in the provided [Scaffold].
+  static void presentInformation(GlobalKey<ScaffoldState> scaffoldKey, String message,
+      {Color iconColor}) {
+    // If the `iconColor` is null, it will default to the system color, whatever
```

Comment:
Which system color do you mean? Is it `Theme.of(context).accentColor` or a different property?

Reply:
Not really sure. Whatever is its default? Haven't investigated exactly which colour it is.

Reply:
It uses the `iconThemeData.color` property to fill the icon colour. I'll update the comment.